docs/linking/4-grant-consent.puml

```diff
@@ -0,0 +1,63 @@
+@startuml
+
+!pragma teoz true
+
+title PISP Linking: Grant consent
+
+participant "PISP" as PISP
+
+box "Mojaloop"
+    participant Switch
+end box
+
+participant "DFSP" as DFSP
+
+autonumber 1 "<b>GRANT-#</b>"
+
+DFSP -> Switch ++: ""POST /consents {""\n\
+    ""  id: "123",""\n\
+    ""  requestId: "456",""\n\
+    ""  initiatorId: "PISP",""\n\
+    ""  participantId: "DFSP",""\n\
+    ""  scopes: {""\n\
```

Comment:
I don't think this data structure will work for us, unless perhaps the _keys_ become arrays of `accountIds`?

ie.
```
scopes: {
  accounts.getBalance: ["12345-67890", "09876-54321"],
  accounts.transfer: ["12345-67890", "09876-54321"]
}
```

Perhaps a better approach would be to have either an array of `Scope` objects:
```
scopes: [
  {
     accountId: "2345-67890",
     scopes: [
       "accounts.getBalance",
       "accounts.transfer"
    ]
  },

  ...
]
```

Reply:
it will be definitely more readable

Reply:
Definitely agree -- done.